packages/relay-test-utils/RelayMockPayloadGenerator.js

```diff
@@ -353,6 +353,77 @@ class RelayMockPayloadGenerator {
     return mockData;
   }
 
+  /**
+   * Generate default enum value
+   * @private
+   */
+  _getCorrectDefaultEnum(
+    enumValues: $ReadOnlyArray<string> | null,
+    value: mixed | Array<mixed>,
+    path: $ReadOnlyArray<string>,
+    applicationName: string,
+  ) {
+    if (enumValues == null) {
+      return value;
+    }
+
+    const enumValuesNormalized = enumValues.map(s => s.toUpperCase());
+
+    if (Array.isArray(value)) {
```

Comment:
I think we can do a little trick here:

```
const valueToValidate = Array.isArray(value) ? value : [value];
```
and the just leave the first part of this if/else statement - where we are validating an array.